### ISSUE TITLE
No Debug mode/Error Warning

### ISSUE THREAD

**Issue by RyanTanWJ** [narrativeandplay/hypedyn2#135]
Users are unable to trace the source of any errors in their Fragement/Node rules/conditions/actions. Consider adding a debug mode or error messages to prompt users to at least be able to run their story, so that they don't have to face a blank screen and wonder what is wrong.

**Comment by lexaay** [comment]
Yes, this is definitely on the to-do list. Mentioned in the old (HypeDyn 1) project here https://blueprints.launchpad.net/hypedyn/+spec/improve-debugging and https://blueprints.launchpad.net/hypedyn/+spec/error-log-for-authors, need to determine when to do this for HypeDyn 2.
